WBooks/Library/LibraryView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LibraryView.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 12/08/21.
+//
+
+import Foundation
+import UIKit
+
+final class LibraryView: NibView {
+    
+    @IBOutlet weak var tableView: UITableView!{
+        didSet{
+            let libraryTableViewCell = LibraryTableViewCell()
```

Comment:
as I mentioned before, you don't need to create this empty cell, just make the nibName and identifier variables static and use them like `LibraryTableViewCell.identifier`

Reply:
Done